contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -142,81 +185,64 @@ library LiquidationLogic {
                 })
             );
 
-        (vars.userTotalDebt, vars.actualDebtToLiquidate) = _calculateDebt(
-            vars.liquidationAssetReserveCache,
-            params
+        (vars.userDebt, vars.actualDebtToLiquidate) = _calculateDebt(
```

Comment:
why are we refactoring the ERC20 liquidation? aave's code has been audited and has been working on mainnet without issues why can't we keep it as is and just add back the close factor?